components/wmtk_components/embedded_component/internal/EmbeddedRemeshing.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <igl/adjacency_list.h>
+#include <igl/edges.h>
+#include <igl/triangle/triangulate.h>
+#include <wmtk/Scheduler.hpp>
+#include <wmtk/TriMesh.hpp>
+
+namespace wmtk::components::internal {
+
+class EmbeddedRemeshing2D
```

Comment:
This class should be named `Embedding` I guess. There is no remeshing happening here.

Also the component should be named `embedding`. File names should be changed accordingly.

Reply:
I changed it to Embedding now. just thought 3D cases could be integrated into the class later, no need for a separate class.